### ISSUE TITLE
Computing the dormant neurons values

### ISSUE THREAD

**Issue by pseudo-rnd-thoughts** [XuGW-Kevin/DrM#4]
Thanks for open sourcing your work

I was looking through your code, in particular, the `cal_dormant_ratio` function.
You compute the absolute value of the forward  pass of observations, why? 
As you are using ReLU with your linear layers then for negative values, these have an activation of 0 not positive given abs

Was this intentional?

If not, this would implement that a larger percentage of your neurons are dormant than the paper suggests

**Comment by XuGW-Kevin** [comment]
Hi, thanks for your question! 
Actually, we used the definition in [The Dormant Neuron Phenomenon in Deep Reinforcement Learning](https://arxiv.org/pdf/2302.12902), which is the paper that "designed" dormant ratio.
I agree that there might be a larger percentage of neurons are dormant than the paper suggests using this definition.
Therefore, in the follow-up work [ACE](https://arxiv.org/pdf/2402.14528), we changed the definition to "gradient", which might solve this issue.
